src/ducks/budgetAlerts/CategoryBudgetNotificationView.js

```diff
@@ -59,7 +59,7 @@ const transformForTemplate = (budgetAlert, t, accountsById, groupsById) => {
   return {
     ...budgetAlert.alert,
     categoryLabel: t(`Data.${type}.${catName}`),
-    currentAmount: (-sumBy(budgetAlert.expenses, tr => tr.amount)).toFixed(2),
+    currentAmount: (-sumBy(budgetAlert.expenses, tr => tr.amount)).toFixed(0),
```

Comment:
Nit: is there a point of using `toFixed(0)`, as opposed to flooring or rounding the number?

Reply:
Not really, I had toFixed(2) before and I changed it to toFixed(0) ;)